daemon/cmd/daemon_main.go

```diff
@@ -1743,6 +1743,12 @@ func initKubeProxyReplacementOptions() {
 		}
 	}
 
+	return
+}
+
+// detectDevicesAndFinishKubeProxyReplacementInit tries to detect bpf_host
+// devices (if needed), and to finish kube-proxy replacement initialization.
+func detectDevicesAndFinishKubeProxyReplacementInit(strict bool) {
```

Comment:
Couldn't we go even further and split this function in two as well? As the long ANDed name indicates, this is `detectDevices()` followed by `finishKubeProxyReplacementInit()`. It would also have the benefit of clearly showing that device detection isn't specific to kube-proxy-replacement.